command/bridge/withdraw/erc721/withdraw_erc721.go

```diff
@@ -0,0 +1,236 @@
+package withdraw
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+
+	"github.com/0xPolygon/polygon-edge/command"
+	"github.com/0xPolygon/polygon-edge/command/bridge/common"
+	cmdHelper "github.com/0xPolygon/polygon-edge/command/helper"
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi"
+	"github.com/0xPolygon/polygon-edge/contracts"
+	"github.com/0xPolygon/polygon-edge/txrelayer"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/spf13/cobra"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+var (
+	wp *common.ERC721BridgeParams = common.NewERC721BridgeParams()
+)
+
+func GetCommand() *cobra.Command {
+	withdrawCmd := &cobra.Command{
+		Use:     "withdraw-erc721",
+		Short:   "Withdraws ERC 721 tokens from the child chain to the root chain",
+		PreRunE: preRun,
+		Run:     run,
+	}
+
+	withdrawCmd.Flags().StringVar(
+		&wp.SenderKey,
+		common.SenderKeyFlag,
+		"",
+		"withdraw transaction sender hex-encoded private key",
+	)
+
+	withdrawCmd.Flags().StringSliceVar(
+		&wp.Receivers,
+		common.ReceiversFlag,
+		nil,
+		"receiving accounts addresses on the root chain",
+	)
+
+	withdrawCmd.Flags().StringSliceVar(
+		&wp.TokenIDs,
+		common.TokenIDsFlag,
+		nil,
+		"tokens ids to send to receiving accounts",
+	)
+
+	withdrawCmd.Flags().StringVar(
+		&wp.PredicateAddr,
+		common.ChildPredicateFlag,
+		contracts.ChildERC721PredicateContract.String(),
+		"ERC 721 child chain predicate address",
+	)
+
+	withdrawCmd.Flags().StringVar(
+		&wp.TokenAddr,
+		common.ChildTokenFlag,
+		contracts.ChildERC721Contract.String(),
+		"ERC 721 child chain token address",
+	)
+
+	withdrawCmd.Flags().StringVar(
+		&wp.JSONRPCAddr,
+		common.JSONRPCFlag,
+		"http://127.0.0.1:9545",
```

Comment:
It was intentionally set this way, because when testing in dev environment, rootchain server JSON RPC is running on a `localhost:8545`, and therefore we've starting JSON RPC endpoint for one of the validator nodes on `localhost:9545`.